docs_src/index.html.md

```diff
@@ -58,8 +58,8 @@ started to address these problems.
 
 This site primarily documents how to set up and use Inertia - to learn more
 about the project, check out our [GitHub repository](https://github.com/ubclaunchpad/inertia)!
-A complete command reference for the Inertia CLI is also available
-[here](/cli).
+A complete command reference for the Inertia CLI is also available [here](/cli).
+If you're interesting in building on Inertia, check out the [API reference](/api).
```

Comment:
Should be interested instead of interesting?

Reply:
Hehe oops good catch